dango/cli/src/main.rs

```diff
@@ -93,25 +94,45 @@ async fn main() -> anyhow::Result<()> {
         });
 
         tracing_subscriber::registry()
+            .with(
+                tracing_subscriber::EnvFilter::try_from_default_env()
+                    .unwrap_or_else(|_| "info".into()), // Default to info if RUST_LOG not set
+            )
             .with(tracing_subscriber::fmt::layer().with_filter(filter))
             .with(sentry_layer())
             .init();
 
         tracing::info!("Sentry initialized");
     } else {
         tracing_subscriber::registry()
-            .with(tracing_subscriber::fmt::layer().with_filter(filter))
+            .with(
+                tracing_subscriber::EnvFilter::try_from_default_env()
+                    .unwrap_or_else(|_| "info".into()), // Default to info if RUST_LOG not set
+            )
+            .with(
+                tracing_subscriber::fmt::layer()
+                    // .with_span_events(tracing_subscriber::fmt::format::FmtSpan::CLOSE),
+            )
```

Comment:
@penso Here you removed the `filter`. It is a custom filter that removes tracing events from actix and async-graphql. Without it, we get a bunch of events on every httpd query, making the logs basically unreadable. Can you add `filter` back?

Reply:
Because it enforces the filtering, which is annoying because I then can't use `RUST_LOG=...` which is standard way to filter out logs so I change log level for a specific crate, and get more logs when debugging.

Reply:
In this case we can delete the custom filter, but you should set up `RUST_LOG` in deployment to hide the actix and async-graphql events. Is this ok?